packages/webapp/src/components/Badge/index.tsx

```diff
@@ -42,33 +47,32 @@ const Badge: React.FC<BadgeProps> = ({ content, title = 'Beta', showIcon = true
 
   const tooltipContent = (
     <div
-      onClick={() => handleTooltipContentHover(true)}
+      onClick={(e) => {
+        e.stopPropagation();
+        handleTooltipContentHover(true);
+      }}
       onMouseEnter={() => handleTooltipContentHover(true)}
       onMouseLeave={() => handleTooltipContentHover(false)}
       className={styles.tooltip}
-    >
-      {content}
-    </div>
+      dangerouslySetInnerHTML={{ __html: content }}
```

Comment:
Could you explain the reason behind this change? Instead of setting the link as HTML within the content string, I think we should keep this as is and set the content up as a React Node including the <a> tag

Reply:
My intentions when developing the component was to make it reusable with different title and content in any other section. So, do not want to hard code anything in the component itself. I will change it to use <Trans> component from 'react-i18next' package(which i found later), is safer way to handle these situations and set the link when passing the content to the <Badge> component. 

Also, it would be nice if you can tell the link address, then i will update that as well.